duckduckgo/iOS
# Add Ukrainian 🇺🇦 localisation

Task/Issue URL: 
Tech Design URL:
CC:

**Description**:
Added Ukrainian localization.



**Steps to test this PR**:
1. Enable the Ukrainian language in Settings.
2. Open the application and check localized strings.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
